android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath 'com.android.tools.build:gradle:8.0.2'
```

Comment:
I think we should increase the agp plugin, I'd prefer to use the newest `8.5.0` if flutter supports it, if not the highest supported version would be great